src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+@import "src/styles/_utils/_variables.scss";
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__wraper {
+    display: flex;
+    width: $box-size * 7;
+    flex-wrap: wrap;
+    gap: 1px; 
+  }
+  
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $box-size;
+    height: $box-size;
+    border: solid black 1px;
```

don't forget to create variables for all colors

```suggestion
    border: solid $c-black 1px;
```